website/src/plugins/remark-npm2yarn.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+// This is a very naive implementation of converting npm commands to yarn commands
+// Works well for our use case since we only use either 'npm install', or 'npm run <something>'
+// Its impossible to convert it right since some commands at npm are not available in yarn and vice/versa
+const convertNpmToYarn = npmCode => {
+  // global install: 'npm i' -> 'yarn'
+  return (
+    npmCode
+      .replace(/^npm i$/gm, 'yarn')
```

Comment:
Why only this line has the ^?

Reply:
npm i => yarn
npm install —save => yarn add

Because if dont have ^ then it can be
yarnnstall —save 😂😂

If we replace the order then its fine but i did it so its explicit if its a “project reinstall”.

Thats why this is such a naive npm2yarn converter. Ive checked there is no such npm pkg to convert

Reply:
npm i => yarn
npm install —save => yarn add

Because if dont have ^ then it can be
yarnnstall —save 😂😂

If we replace the order then its fine but i did it so its explicit if its a “project reinstall”.

Thats why this is such a naive npm2yarn converter. Ive checked there is no such npm pkg to convert